cluster/gce/coreos/node.yaml

```diff
@@ -123,7 +123,7 @@ coreos:
         ExecStartPre=/usr/bin/mkdir -p /etc/kubernetes/manifests
         ExecStartPre=/run/setup-auth.sh
         ExecStart=/opt/kubernetes/server/bin/kubelet \
-        --api_servers=https://kubernetes-master.c.${PROJECT_ID}.internal \
+        --api_servers=https://${KUBERNETES_MASTER_NAME}.c.${PROJECT_ID}.internal \
```

Comment:
There is a `MASTER_NAME` in the config, should we use that instead?

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/gce/config-default.sh#L36